glean-core/src/metrics/mod.rs

```diff
@@ -99,11 +103,12 @@ impl Metric {
             Metric::Counter(_) => "counter",
             Metric::Datetime(_, _) => "datetime",
             Metric::Experiment(_) => panic!("Experiments should not be serialized through this"),
+            Metric::Quantity(_) => "quantity",
             Metric::String(_) => "string",
             Metric::StringList(_) => "string_list",
-            Metric::Uuid(_) => "uuid",
             Metric::Timespan(..) => "timespan",
             Metric::TimingDistribution(..) => "timing_distribution",
+            Metric::Uuid(_) => "uuid",
```

Comment:
Hm, moved just to make this alphabetically sorted?

Reply:
Yep.